docs/snippets/gettingStarted/webApi/simpleApi.ts

```diff
@@ -139,6 +141,40 @@ export const shoppingCartApi =
         return NoContent();
       }),
     );
+
+    // Get Shopping Cart
+    router.get(
```

Comment:
**HINT:** The method as a test method is okay as it is. You could consider providing the customised evolve method that would already build the desired read model without the need to do the additional mapping. But that's a nice to have if you wanted to play more with event sourcing. This would be a dedicated `ShoppingCartDetails` type with is own `evolve` function.